plasma_framework/contracts/src/exits/utils/ExitId.sol

```diff
@@ -2,68 +2,58 @@ pragma solidity 0.5.11;
 
 import "../../utils/Bits.sol";
 import "../../utils/PosLib.sol";
+import "./OutputId.sol";
 
 library ExitId {
     using PosLib for PosLib.Position;
-    using Bits for uint160;
+    using Bits for uint168;
     using Bits for uint256;
 
+    uint8 constant private FIRST_BIT_LOCATION = 167;
+
     /**
      * @notice Checks whether exitId is a standard exit ID
      */
-    function isStandardExit(uint160 _exitId) internal pure returns (bool) {
-        return _exitId.getBit(151) == 0;
+    function isStandardExit(uint168 exitId) internal pure returns (bool) {
+        return exitId.getBit(FIRST_BIT_LOCATION) == 0;
     }
 
     /**
      * @notice Given transaction bytes and UTXO position, returns its exit ID
      * @dev Computation of a deposit ID is different to any other tx because txBytes of a deposit tx can be a non-unique value
-     * @notice Output index must be within range 0 - 255
-     * @param _isDeposit Defines whether the tx for the exitId is a deposit tx
-     * @param _txBytes Transaction bytes
-     * @param _utxoPos UTXO position of the exiting output
-     * @return _standardExitId Unique ID of the standard exit
+     * @param isDeposit Defines whether the tx for the exitId is a deposit tx
+     * @param txBytes Transaction bytes
+     * @param utxoPos UTXO position of the exiting output
+     * @return standardExitId Unique ID of the standard exit
      *     Anatomy of returned value, most significant bits first:
-     *     8-bits - output index
      *     1-bit - in-flight flag (0 for standard exit)
-     *     151-bits - hash(tx) or hash(tx|utxo) for deposit
+     *     167-bits - hash(tx) or hash(tx|utxo) for deposit
      */
     function getStandardExitId(
-        bool _isDeposit,
-        bytes memory _txBytes,
-        PosLib.Position memory _utxoPos
+        bool isDeposit,
+        bytes memory txBytes,
+        PosLib.Position memory utxoPos
     )
         internal
         pure
-        returns (uint160)
+        returns (uint168)
     {
-        if (_isDeposit) {
-            bytes32 hashData = keccak256(abi.encodePacked(_txBytes, _utxoPos.encode()));
-            return _computeStandardExitId(hashData, _utxoPos.outputIndex);
+        bytes32 outputId;
+        if (isDeposit) {
+            outputId = OutputId.computeDepositOutputId(txBytes, utxoPos.outputIndex, utxoPos.encode());
+        } else {
+            outputId = OutputId.computeNormalOutputId(txBytes, utxoPos.outputIndex);
         }
 
-        return _computeStandardExitId(keccak256(_txBytes), _utxoPos.outputIndex);
+        return uint168((uint256(outputId) >> (256 - FIRST_BIT_LOCATION)));
```

Comment:
Does this functionally still return the same exit id? Is this meant as a code clean up?

Reply:
Yes.
Previously `_computeStandardExitId()` was called twice.
Now that code is one line that's not repeated, so no need to split it out into a separate function